Source/JavaScriptCore/Scripts/process-entitlements.sh

```diff
@@ -97,16 +97,20 @@ function maccatalyst_process_testapi_entitlements()
 
 function ios_family_process_jsc_entitlements()
 {
-    plistbuddy Add :com.apple.private.verified-jit bool YES
-    plistbuddy Add :dynamic-codesigning bool YES
+    if [[ "${WK_PLATFORM_NAME}" != watchos ]]; then
+        plistbuddy Add :com.apple.private.verified-jit bool YES
+        if (( "${WK_PLATFORM_NAME}" == iphoneos && $(( ${SDK_VERSION_ACTUAL} )) >= 170400
+            || "${WK_PLATFORM_NAME}" == appletvos && $(( ${SDK_VERSION_ACTUAL} )) >= 170400
+            || "${WK_PLATFORM_NAME}" == xros && $(( ${SDK_VERSION_ACTUAL} )) >= 10100 ))
+        then
+            plistbuddy Add :com.apple.security.cs.allow-jit bool YES
+        else
```

Comment:
Should we keep the dynamic codesigning entitlement for some time longer, or is it safe to remove?